### ISSUE TITLE
bug: members dropdown in CoreStandardGroup add member form is not taking namespace of Nodes into account

### ISSUE THREAD

**Issue by wvandeun** [opsmill/infrahub#3457]
### Component

Frontend UI

### Current Behavior

When you open the add member form of a CoreStandardGroup, you first have to select a Nodekind for the member you want to add. The dropdown listen all the kinds of nodes, seems to only display the name and not the combination of namespace + name.

This is a problem when you have a 2 nodes with the same name, in a different namespace. But it is also inconsistent with how we do this in other places.

### Expected Behavior

The namespace + name should be used when we prompt the user for a node kind in the add member form.

### Steps to Reproduce

- start instance of Infrahub
- load this schema
```yaml
---
version: "1.0"
nodes:
  - name: Tag
    namespace: Test
    attributes:
      - name: name
        kind: Text
``` 
- create a group
- open the detailed view of the group
- open the members tab
- click the add button
- the dropdown will have `Tag` listed two times, without the namespace you have no idea which Tag you need to select.

**Comment by BeArchiTek** [comment]
It is fixed in 0.16.

It got fixed when we changed the profile's label in those forms. Right now both name and namespace are indicated
![image](https://github.com/user-attachments/assets/e505ff87-8af9-4243-9b5b-5a76a0ce5870)

The other issue was https://github.com/opsmill/infrahub/issues/3681

**Comment by wvandeun** [comment]
Seems to be closed in 0.16.0
